src/components/ProductGallery/ProductGallery.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../utils/mixins.scss';
+@import '../../utils/variables.scss';
+
+.gallery {
+  display: flex;
+  @include for-phone() {
+    flex-direction: column-reverse;
+  }
+
+  &_allImages {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    width: 80px;
+
+    @include for-phone() {
+      flex-direction: row;
+      width: 100%;
+      justify-content: center;
+      margin-top: 16px;
+    }
+
+    &_item {
+      object-fit: contain;
+      border: 1px solid $color-elements;
+
+      @include for-desktop() {
+        height: 80px;
+        width: 80px;
+      }
+      @include for-tablet() {
+        height: 35px;
+        width: 35px;
+      }
+      @include for-phone() {
+        height: 49px;
+        width: 52px;
+      }
+
+      &:hover {
+        border: 1px solid $color-primary;
+      }
```

Comment:
Could this border be also visible on active photo?

Reply:
Yes it should be, I will add fix in task branch which i'm currently doing.